bin/cron/activity-monitor

```diff
@@ -192,31 +183,22 @@ def excluded_browser?(ua_string)
 end
 
 def main
-  now = DateTime.now.new_offset(0)
-
-  hours = HoursToTest
-  finish = DateTime.new(now.year, now.month, now.day, now.hour, now.minute).new_offset(0).strftime('%Q').to_i
-  start = finish - (hours * OneHour)
+  finish = DateTime.now.new_offset(0).strftime('%s') * SECONDS_PER_MSEC
```

Comment:
Note that `strftime('%Q')` seems to return milliseconds (avoiding the need for the conversion). That said, I'm avoiding its usage since the implementation and [documentation](http://apidock.com/ruby/v1_9_3_125/DateTime/strftime) [disagree](http://apidock.com/ruby/DateTime/strftime#1420-Bug-in-Ruby-or-this-documentation).